docs/development/extensions-core/mysql.md

```diff
@@ -41,6 +41,19 @@ This should fetch a JAR file named similar to 'mysql-connector-java-x.x.xx.jar'.
 
 Copy or symlink this file inside the folder `extensions/mysql-metadata-storage` under the distribution root directory.
 
+## Alternative: Installing the MariaDB connector library
+
+THis extension also supports using the MariaDB connector jar, though it is also not included in the Druid distribution and must be installed separately like the MySQL connector.
+
+- Download from the MariaDB site: https://mariadb.com/downloads/connector
+- Maven Central: https://repo1.maven.org/maven2/org/mariadb/jdbc/mariadb-java-client/2.7.3/mariadb-java-client-2.7.3.jar
+
+This should fetch a JAR file named similar to 'maria-java-client-x.x.x.jar'.
+
+Copy or symlink this file to `extensions/mysql-metadata-storage` under the distribution root directory.
+
+In Druid properties, `druid.metadata.mysql.driver.driverClassName=org.mariadb.jdbc.Driver` must be set, which will instruct the `mysql-metadata-storage` extension to use the MariaDB connector library instead of MySQL. Depending on the MariaDB client library version, it will support both `jdbc:mysql:` and `jdbc:mariadb:` connection URIs, though the parameters to configure the connection vary between implementations, so be sure to [check the documentation](https://mariadb.com/kb/en/about-mariadb-connector-j/#connection-strings) for details.
```

Comment:
```suggestion
To configure the `mysql-metadata-storage` extension to use the MariaDB connector library instead of MySQL,  set the following in the Druid properties: `druid.metadata.mysql.driver.driverClassName=org.mariadb.jdbc.Driver` 
Depending on the MariaDB client library version, the connector supports both `jdbc:mysql:` and `jdbc:mariadb:` connection URIs. However, the parameters to configure the connection vary between implementations, so be sure to [check the documentation](https://mariadb.com/kb/en/about-mariadb-connector-j/#connection-strings) for details.
```